package.json

```diff
@@ -62,11 +62,12 @@
     "react-native-incall-manager": "3.3.0",
     "react-native-iphone-x-helper": "1.3.1",
     "react-native-keyboard-spacer": "0.4.1",
-    "react-native-linear-gradient": "2.5.6",
+    "react-native-linear-gradient": "^2.5.6",
```

Comment:
Remove the "^" character from the front as it will keep on updating the version any time npm install is run.